spec/lib/evss/disability_compensation_form/data_translation_all_claim_spec.rb

```diff
@@ -100,7 +100,138 @@
+      end
+    end
+
+    describe 'form 0781/a' do
+      context 'when a form 0781/a is included with Form 526' do
+        let(:form_json) do
+          File.read('spec/support/disability_compensation_form/submissions/with_0781.json')
+        end
+
+        context 'when the form526_include_document_upload_list_in_overflow_text flipper is enabled' do
+          before do
+            Flipper.enable(:form526_include_document_upload_list_in_overflow_text)
+          end
+
+          it 'includes a note in the overflow text' do
+            expected_note = "VA Form 0781/a has been completed by the applicant and sent to the VBMS eFolder\n"
+            expect(described_class.new(user, JSON.parse(form_json), false).send(:overflow_text)).to eq(expected_note)
+          end
+        end
+
+        context 'when the form526_include_document_upload_list_in_overflow_text flipper is disabled' do
+          before do
+            Flipper.disable(:form526_include_document_upload_list_in_overflow_text)
+          end
+
+          it 'does not include a note in the overflow text' do
+            expect(subject.send(:overflow_text)).to eq('')
+          end
+        end
+      end
+
+      context 'when a form 0781/a is not included with Form 526' do
+        before do
+          Flipper.enable(:form526_include_document_upload_list_in_overflow_text)
+        end
+
+        let(:form_json) do
+          File.read('spec/support/disability_compensation_form/submissions/only_526.json')
+        end
+
+        it 'does not include a note in the overflow text' do
+          expect(subject.send(:overflow_text)).to eq('')
+        end
+      end
+    end
+
+    describe 'veteran uploaded document list' do
+      subject { described_class.new(user, form_content, false) }
+
+      context 'when the veteran has uploaded documents to support the claim' do
+        let(:file1_guid) { SecureRandom.uuid }
+        let(:file2_guid) { SecureRandom.uuid }
+        let(:form_content) do
+          {
+            'form526' => {
+              'attachments' => [
+                { 'confirmationCode' => file1_guid },
+                { 'confirmationCode' => file2_guid }
+              ]
+            }
+          }
+        end
+
+        let!(:file1) do
+          create(
+            :supporting_evidence_attachment,
+            guid: file1_guid,
+            file_data: { filename: 'my_file_1.pdf' }.to_json
+          )
+        end
+
+        let!(:file2) do
+          create(
+            :supporting_evidence_attachment,
+            guid: file2_guid,
+            file_data: { filename: 'my_file_2.pdf' }.to_json
+          )
+        end
+
+        context 'when the form526_include_document_upload_list_in_overflow_text flipper is enabled' do
+          before do
+            Flipper.enable(:form526_include_document_upload_list_in_overflow_text)
+          end
+
+          context 'when there is less than 40 files present' do
```

Comment:
when there ~is~ **are** less

Reply:
This test was removed when I changed the behavior